include/celero/Print.h

```diff
@@ -41,6 +41,7 @@ namespace celero
 		void Failure(const std::string& x);
 		void Run(std::shared_ptr<Result> x);
 		void Done(std::shared_ptr<Result> x);
+		void DoneExperiment(std::shared_ptr<Experiment> x);
```

Comment:
Thanks for submitting your work.  My worry here is that by displaying the total run time for an experiment, the user may think this is a measurement of some significance.  In fact, it would be all-encompassing and include time spent doing things OTHER than the experiment itself.